Index: www/applications/vanilla/models/class.draftmodel.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- www/applications/vanilla/models/class.draftmodel.php	(revision )
+++ www/applications/vanilla/models/class.draftmodel.php	(revision )
@@ -163,6 +163,9 @@
             unset($FormPostValues['Closed']);
         }
 
+        //	Prep and fire event
+        $this->EventArguments['FormPostValues'] = &$FormPostValues;
+        $this->FireEvent('BeforeSaveDraft');
         if (ArrayValue('Sink', $FormPostValues, '') === false) {
             unset($FormPostValues['Sink']);
         }
\ No newline at end of file
